docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -83,6 +83,13 @@ View the generated files and/or error messages with:
 /usr/lib/systemd/system-generators/podman-system-generator {--user} --dryrun
 ```
 
+#### Debugging a limited set of unit files
+
+If you would like to debug a limited set of unit files, you can copy them to a separate directory and set the
+`QUADLET_UNIT_DIRS` environment variable to this directory when running the command below.
```

Comment:
> the command below.

I am not sure which command is being referred to.

Reply:
You're right. When I first wrote it I meant above but wrote below. I've kept below and added the command. I hope this is better